x-pack/qa/core-rest-tests-with-security/build.gradle

```diff
@@ -15,9 +15,6 @@ integTestRunner {
     ['cat.aliases/10_basic/Empty cluster',
      'index/10_with_id/Index with ID',
      'indices.get_alias/10_basic/Get alias against closed indices',
-     'cat.templates/10_basic/No templates',
```

Comment:
These are skipped by the `skip` section in the test now which I find *much* easier to read.